config/overlays/openshift/kustomization.yaml

```diff
@@ -9,6 +9,7 @@ resources:
 #- ../certmanager
 # [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
 - ../../prometheus
+- size-calculator
```

Comment:
The size-calculator should not be part of this kustomization as per adding this line here makes the calculators manifest part of the operator bundle. However, the bundle is dedicated to the operator. I suggest to remove this line and enable the size-calculator bundle with a separate makefile target as described above, e.g.:

```makefile
deploy-size-calculator:  $(KUSTOMIZE) ## Deploy storage size calculator
	$(KUSTOMIZE) build config/overlays/openshift/storage-size-calculator | kubectl apply -f -

undeploy-size-calculator: ## Undeploy storage size calculator
	$(KUSTOMIZE) build config/overlays/openshift/storage-size-calculator | kubectl delete -f -
```